sockeye/training.py

```diff
@@ -587,6 +588,18 @@ def _load_trainer_states(self, fname):
         self.trainer.load_states(fname)
         logger.info('Loaded optimizer states from "%s"', fname)
 
+    def _save_lr_scheduler(self, fname):
+        with open(fname, "wb") as fp:
+            pickle.dump(self.trainer.optimizer.lr_scheduler, fp)
+        logger.info('Saved lr_scheduler to "%s"', fname)
```

Comment:
can we combine / simplify to one line of logging instead?
```suggestion
        logger.info('Saved '%s' to "%s", self.trainer.optimizer.lr_scheduler, fname)
```

Reply:
ok